app/auth/components/SignupForm.tsx

```diff
@@ -5,24 +5,30 @@ import { Signup } from "app/auth/validations"
 
 import { Box, Flex, Heading, Text } from "@chakra-ui/layout"
 import { Input } from "@chakra-ui/input"
-import { Button } from "@chakra-ui/button"
 import { Field } from "formik"
+import { Center, useToast } from "@chakra-ui/react"
 
 type SignupFormProps = {
   onSuccess?: () => void
 }
 
 export const SignupForm = (props: SignupFormProps) => {
-  const [signupMutation] = useMutation(signup)
+  const toast = useToast()
+
+  const [signupMutation] = useMutation(signup, {
+    onSuccess: (result) => {
+      if (result && !result?.notify?.success) {
+        toast({
```

Comment:
just a reminder to set these up in Knock instead of silently failing on these notifications